powershell-yaml.psm1

```diff
@@ -38,19 +38,43 @@ function Convert-ValueToProperType {
         if (!($Value -is [string])) {
             return $Value
         }
-        $culture = New-Object System.Globalization.CultureInfo("invariant")
```

Comment:
The reason for forcing the ```invariant``` culture was that regional settings affect conversion. See issue: https://github.com/cloudbase/powershell-yaml/issues/8

Essentially, the thousands separator and the decimal separator are culture specific. The ```invariant``` culture uses the dot as a decimal separator, which is what the YAML spec demands IIRC.